src/components/markdown/partial-styles/_img.scss

```diff
@@ -1,3 +1,4 @@
 img {
     max-width: 100%;
+    border-radius: 0.25rem;
```

Comment:
this wasn't stricktly necessarily or related to this PR. But since everything else is a `chore` I wouldn't get a release without a `fix`